deeprank/parse/param.py

```diff
@@ -0,0 +1,37 @@
+import re
+import logging
+
+
+_log = logging.getLogger(__name__)
+
+class VanderwaalsParam:
+    def __init__(self, epsilon, sigma):
+        self.epsilon = epsilon
+        self.sigma = sigma
+
+
+class ParamParser:
+
+    LINE_PATTERN = re.compile(r"^NONBonded\s+([A-Z0-9]{1,4})((\s+\-?[0-9]+\.[0-9]+){4})\s*$")
+
+    @staticmethod
+    def parse(file_):
+        result = {}
+        for line in file_:
+            if line.startswith('#') or len(line.strip()) == 0:
+                continue
+
+            m = ParamParser.LINE_PATTERN.match(line)
+            if not m:
+                raise ValueError("unmatched param line: {}".format(repr(line)))
+
+            atom_type = m.group(1)
+            if atom_type in result:
+                raise ValueError("duplicate atom type: {}".format(repr(atom_type)))
+
+            number_strings = m.group(2).split()
+            epsilon = float(number_strings[0])
+            sigma = float(number_strings[1])
```

Comment:
We would need the last two columns from "protein-allhdg5-4_new.param" file for intra-chain interactions.

Reply:
Not in this parser, but I could add the values to the VanderwaalsParam object. What shall we call them? epsilon2 and sigma2?

(I didn't see these column values used in the original AtomicFeature class)

Reply:
Yes, I think that would be fine. 
The first two column values are for inter-chain and the last column values are for intra-chain in the param file. So we can have the last two column values as epsilon2 and sigma2

Reply:
we could also call them intra_epsilon and intra_sigma then. Does this mean I should also use these values in the vanderwaals computation, wherever two atoms are from the same chain?

Reply:
Yes, that's correct. 
In the event of multi-chain input structure we can have inter-chain as well (but this part need not be prioritized). 

The intra_ and inter_ in code would also add clarity. 
inter_ : first two columns, intra_ : last two.

Reply:
Yes, this is perfect.
Thank you.